### PR TITLE
fix: fix log line on consistency errors

### PR DESCRIPTION
**Pull request by jigisha620**
Fixes #N/A 

**Description**
Fixes the error message for consistency failures

**How was this change tested?**

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by njtran — COMMENTED** [review]
/lgtm
/approve

File: pkg/controllers/nodeclaim/consistency/controller.go [hunk 1]
```diff
@@ -101,7 +102,7 @@ func (c *Controller) Reconcile(ctx context.Context, nodeClaim *v1beta1.NodeClaim
 			return reconcile.Result{}, fmt.Errorf("checking node with %T, %w", check, err)
 		}
 		for _, issue := range issues {
-			log.FromContext(ctx).Error(err, "consistency error")
+			log.FromContext(ctx).Error(errors.New(string(issue)), "consistency error")
```
Comment (njtran) [inline pkg/controllers/nodeclaim/consistency/controller.go:105]: I'm guessing we have to change this on main to fit the linting changes too, right?

**Review by njtran — COMMENTED** [review]
/lgtm

File: pkg/controllers/nodeclaim/consistency/controller.go (hunk 1 above)
Comment (jigisha620) [reply pkg/controllers/nodeclaim/consistency/controller.go:105]: This change is already there on main.

**Pull request merged**